sql/policy_api.sql

```diff
@@ -90,7 +90,8 @@ CREATE OR REPLACE FUNCTION @extschema@.add_continuous_aggregate_policy(
     timezone TEXT = NULL,
     include_tiered_data BOOL = NULL,
     buckets_per_batch INTEGER = NULL,
-    max_batches_per_execution INTEGER = NULL
+    max_batches_per_execution INTEGER = NULL,
+    refresh_newest_first BOOL = NULL
```

Comment:
This is a matter of choice, but since setting this option is typically done to change the _existing_ behavior, it might be more intuitive to think of this as if you are "turning ON" a different behavior:

`refresh_oldest_first =>true`

In other words, if you don't specify this option it is off and the default behavior applies.

Either one works, of course, just pointing out the alternative.

Reply:
I got your point and actually I did it in the first attempt, but those optional policy options are all NULL by default to don't be included it into the JSONB config stored in the jobs table. So it is included only when the user explicity set it in the API.

You can see [in regression tests](https://github.com/timescale/timescaledb/pull/7915/files#diff-043595e34db210025268e34eb464f4b98da7b0e52815e7d8f53a3e6e709c0d06R631) that the optional options are included only when explicity used in the API.